### PR TITLE
chore: Javadoc fixes

### PR DESCRIPTION
**Pull request by Ansku**

### CODE REVIEW

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/ElementQuery.java [hunk 1]
```diff
@@ -279,6 +295,8 @@ public ElementQuery<T> context(SearchContext searchContext) {
     /**
      * Return the {@link SearchContext} previously set with the {@link #in}
      * call.
+     *
+     * @return context
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/ElementQuery.java:299]: Doesn't really add much information

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/HasTestBenchCommandExecutor.java [hunk 2]
```diff
@@ -23,8 +23,8 @@ public interface HasTestBenchCommandExecutor extends HasSearchContext {
     /**
      * Return a reference to the related {@link TestBenchCommandExecutor}
      * instance.
-     * 
-     * @return
+     *
+     * @return executor
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/HasTestBenchCommandExecutor.java:27]: Doesn't add much information

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/Parameters.java [hunk 3]
```diff
@@ -158,6 +160,7 @@ public static boolean isScreenshotComparisonCursorDetection() {
      * Sets the directory to search for reference images.
      *
      * @param screenshotReferenceDirectory
+     *            directory path
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/Parameters.java:163]: This adds SOME information, but perhaps more would be better?

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/Parameters.java [hunk 4]
```diff
@@ -175,6 +178,7 @@ public static String getScreenshotReferenceDirectory() {
      * Sets the directory where error screen shots are stored.
      *
      * @param screenshotErrorDirectory
+     *            directory path
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/Parameters.java:181]: See above

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/Parameters.java [hunk 5]
```diff
@@ -223,8 +230,10 @@ public static void setCaptureScreenshotOnFailure(
     }
 
     /**
-     * @return whether to capture a screen shot when a test fails or not.
+     * @return whether to capture a screen shot when a test fails or not
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/Parameters.java:233]: This should probably have a slightly lengthier description, as above

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/TestBench.java [hunk 6]
```diff
@@ -119,6 +119,8 @@ public static TestBenchElement createElement(WebElement webElement,
      * TestBenchCommandExecutor. This feature is advanced and potentially
      * dangerous.
      *
+     * @param <T>
+     *            the type of the class
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/TestBench.java:123]: As a description, this is confusing. "Type of the class"? Should it perhaps inherit from something or have special traits?

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/screenshot/ImageUtil.java [hunk 7]
```diff
@@ -70,7 +70,8 @@ public static String encodeImageToBase64(BufferedImage image) {
      * Get luminance value for the given rgb value.
      * 
      * @param rgb
-     * @return
+     *            rgb value
```
Comment (thevaadinman) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/screenshot/ImageUtil.java:73]: I get the feeling that we might want to explain that the red, green and blue byte values should be packed into an integer at given offsets somewhere here..

**Review by thevaadinman — COMMENTED** [review]
Some comments, but in general this seems fine

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/screenshot/ImageUtil.java (hunk 7 above)
Comment (thevaadinman) [reply vaadin-testbench-core/src/main/java/com/vaadin/testbench/screenshot/ImageUtil.java:73]: something like `The RGB value is assumed to be taking up the low 24 bits of the integer sample, with red shifted by 16 bits, green by 8 bits and blue by 0 bits, ORed together. This function will not return the correct values for other channel orders.`

**Review by thevaadinman — APPROVED** [review]

**Comment by Ansku** [comment]
Added some tweaks, left out the 'no added value' changes because the main purpose is to get rid of the Javadoc errors in build logs and the method description already contains the extended information.

**Review by thevaadinman — APPROVED** [review]

**Pull request merged**